nft-lvlup/.env.example

```diff
@@ -58,3 +58,6 @@ DB_PORT="5432"
 # Note: This is the address of the native nft sale proxy contract
 # Example: "0xbD9e6bA880d6302A0B93456308a5A998Ffd8eb5E"
 NATIVE_NFT_SALE_PROXY=""
+# Fill in in order to use nft sale frontend admin functionality
```

Comment:
Depends what contract you want to modify :) I think for volcaneers it will be `0xa335d662BB47409e04F06dC7Fd03cEc854530172` for the NFT, not sure about the other. 

When you deploy your own these are written down in the output

Reply:
if you just wanted to test with some contract, I used these:
NATIVE_NFT_SALE_PROXY=0xbD9e6bA880d6302A0B93456308a5A998Ffd8eb5E
NFT=0xb1D0c3142Ef25d7A693bD60b6892a0Eea27eFAf0
NFT_SALE_PROXY=0xAA903650374016822219f6B2a59098097Ebd88D1